tools/storyteller/StoryWriter.java

```diff
@@ -148,7 +148,11 @@ private String getAbsUnsharedStoriesFolderPath() {
     return Storyteller.getUnsharedStoriesAbsPath(config);
   }
 
-  private String getId() {
+  /* Returns the first element of UUID.
+   * UUID represents a 128-bit long value that is unique.
+   * It is represented in 32 Hexadecimal characters and is mixed with 4 dashes characters.
```

Comment:
`s/and is mixed with 4 dashes characters/separated by 4 dashes`

Reply:
Done.